server/aws/modules/metrics/create_csv.tf

```diff
@@ -0,0 +1,114 @@
+
+resource "aws_lambda_function" "unmasked_metrics" {
+  function_name = "unmasked_metrics"
+
+  image_uri = var.create_csv_image
+
+  runtime = var.lambda_function_runtime
+  role    = aws_iam_role.backoff.arn
+
+  vpc_config {
+    security_group_ids = [aws_security_group.metrics_csv_sg.id]
+    subnet_ids         = var.subnet_ids
+  }
+
+  environment {
+    variables = {
+      MASK_DATA = false
```

Comment:
Needs a `BUCKET_NAME` environment variable, which is read here: https://github.com/cds-snc/metrics-debug-covid-alert/blob/f1bbe976d0e179f62b932bc6f1f09ec71bf8224e/save_to_bucket.py#L7 should be able to get that using the following: 

```hcl
BUCKET_NAME = aws_s3_bucket.unmasked_metrics.bucket.id
```